docs/schema/changelog.rst

```diff
@@ -37,6 +37,7 @@ Changed
 - Do not reference codelists in ``statementType`` - fixes issues caused by the way we use this field to select which subschema to use for validation. This should have no change for anyone using the compiled schema but may affect anyone using the files in ``schema/`` directly.
 - `interestLevel` renamed to `directOrIndirect`.
 - The ``interestType`` codelist codes have had `OfTrust` removed so they refer to any type of legal arrangement. Their descriptions have been edited to reflect these changes. `beneficiaryOfTrust` has been changed to `beneficiaryOfLegalArrangement` to avoid ambiguity and `otherInfluenceOrControlOfTrust` has been removed.
+- The interestType codelist has had the following new codes added: `controlViaCompanyRulesOrArticles`, `controlByLegalFramework`, `boardMember`, `boardChair`, `unknownInterest`, `unpublishedInterest`, `enjoymentAndUseOfAssets`, `rightToProfitOrIncomeFromAssets`
```

Comment:
There should be double backquotes around interestType here. (The whole changelog will get an edit before release, though. So feel free to merge without making this edit.)

Reply:
I've added it anyway. Seeing as I made you add a four-letter word earlier!